src/hotspot/share/opto/callnode.cpp

```diff
@@ -1080,12 +1094,32 @@ void CallRuntimeNode::dump_spec(outputStream *st) const {
   CallNode::dump_spec(st);
 }
 #endif
+uint CallLeafVectorNode::size_of() const { return sizeof(*this); }
+uint CallLeafVectorNode::cmp( const Node &n ) const {
+  CallLeafVectorNode &call = (CallLeafVectorNode&)n;
+  return CallLeafNode::cmp(call) && _num_bits == call._num_bits;
+}
```

Comment:
It would be better place these two functions CallLeafVectorNode::cmp() and CallLeafVectorNode::size_of() after CallLeafNode::dump_spec() instead of CallRuntimeNode::dump_spec()